src/ServiceInsight/Framework/Modules/CoreModule.cs

```diff
@@ -21,7 +21,7 @@ protected override void Load(ContainerBuilder builder)
             builder.RegisterType<HeaderContentDecoder>().As<IContentDecoder<IList<HeaderInfo>>>();
             builder.RegisterType<NetworkOperations>().SingleInstance();
             builder.RegisterType<AppLicenseManager>().SingleInstance();
-            builder.RegisterType<ServiceControlConnectionProvider>().InstancePerLifetimeScope();
+            builder.RegisterType<ServiceControlConnectionProvider>().SingleInstance();
```

Comment:
Just wondering...isn't InstancePerLifetimeScope essentially the same as singleton here? Since we only have one container that is never disposed, effectively InstancePerLifetimeScope would give us singletons...anything I'm missing?